docs/querying-data.rst

```diff
@@ -11,15 +11,15 @@ All of the examples in this page refer to a client object, which can
 be initialized as follows:
 ::

```

Comment:
Can still provide the language hint, like `.. code-block:: python` instead of the generic `::`

https://www.sphinx-doc.org/en/1.5/markup/code.html#directive-code-block

Reply:
it is python by default
`The valid values for the highlighting language are:

none (no highlighting)
python (the default when highlight_language isn’t set)`

should specify if otherwise

Reply:
pushed change to update each caller to epxlicit python blocks